tests/system/components/authHeader.js

```diff
@@ -106,4 +107,35 @@ module.exports = {
         .assert.attributeContains('section > div > div > div > div > div:nth-child(4) > ul > li:nth-child(2) > a', "href", "help")
         .end();
     }
+
+    'Verify Account Nav can Be toggeled Open and closed': function(browser) {
+        browser
+        .click(selectors.headerProfileBtn)
+        .waitForAnimation()
+        .verify.elementsVisible(
+            selectors.headerAccountNav,
+        )
+        .click(selectors.headerProfileBtn)
+        .verify.hidden(selectors.headerAccountNav)
+        .end();
+    },
+
+    'Verify Account Nav switch to account switcher and back': function(browser) {
+        browser
+        .click(selectors.headerProfileBtn)
+        .waitForAnimation()
+        .verify.elementsVisible(
+            selectors.headerAccountNav,
+            selectors.headerAccountNavSwitchAccountButton,
+            selectors.headerAccountNavGreeting,
+            selectors.headerAccountNavBalanceSummary,
+            selectors.headerAccountNavSettingsLinks
+        )
+        .click(selectors.headerAccountNavSwitchAccountButton)
+        .waitForAnimation()
+        .verify.elementsVisible(selectors.headerAccountSwitcher)
+        .click(selectors.headerAccountSwitcherCancelButton)
+        .verify.elementsVisible(selectors.headerAccountNav)
+        .end();
+    },
```

Comment:
This comma isn't necessary because it's the last test in the file. Putting a comma here tells nightwatch to expect another test and throws a js error.